baseledger/x/bridge/types/params.go

```diff
@@ -19,16 +26,38 @@ func NewParams() Params {
 
 // DefaultParams returns a default set of parameters
 func DefaultParams() Params {
-	return NewParams()
+	return Params{
+		WorktokenEurPrice: "0.1",
+	}
 }
 
 // ParamSetPairs get the params.ParamSet
 func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
-	return paramtypes.ParamSetPairs{}
+	return paramtypes.ParamSetPairs{
+		paramtypes.NewParamSetPair(ParamsStoreKeyWorktokenEurPrice, &p.WorktokenEurPrice, validateWorktokenEurPrice),
+	}
+}
+
+// Equal returns a boolean determining if two Params types are identical.
+func (p Params) Equal(p2 Params) bool {
+	bz1 := ModuleCdc.MustMarshalLengthPrefixed(&p)
+	bz2 := ModuleCdc.MustMarshalLengthPrefixed(&p2)
+	return bytes.Equal(bz1, bz2)
 }
 
 // Validate validates the set of params
 func (p Params) Validate() error {
+	if err := validateWorktokenEurPrice(p.WorktokenEurPrice); err != nil {
+		return sdkerrors.Wrap(err, "worktoken eur price")
+	}
+
+	return nil
+}
+
+func validateWorktokenEurPrice(i interface{}) error {
+	if _, err := sdk.NewDecFromStr(fmt.Sprint(i)); err != nil {
```

Comment:
should we use this to add some basic validation, like price can not be 0 or something like that?

Reply:
will add